tests/model_registry/rbac/utils.py

```diff
@@ -178,3 +181,16 @@ def assert_forbidden_access(endpoint: str, token: str) -> None:
     except ForbiddenException:
         # This is what we want - access is properly forbidden
         pass
+
+
+def should_skip_rbac_tests() -> bool:
```

Comment:
Is it possible to use a fixture? A fixture would not require to determine if it is BYOIDC again and again, but util would.

Reply:
Unfortunately not, to be used in the `skipif` mark we need it to not be a fixture or rely on fixtures unless we want to be hacky because it needs to run at collection time. In the worst case scenario it only runs 2 times though.